omod/src/main/resources/apps/smartSearch_app.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "smart.search",
+    "instanceOf": "coreapps.template.findPatient",
+    "description": "Basic patient search, by ID or Name (using OpenMRS's standard patient search)",
+    "config": {
+      "afterSelectedUrl": "/ms/smartPatientSelected?patientId={{patientId}}"
+    }
+  }
+]
```

Comment:
@theanandankit Could you please remove this code from this PR and make a new PR for it.

Reply:
@VaishSiddharth thanks for the review, i will make another new PR for that.